NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# Sb product details

# Description

Users can now add items to cart from the products details page, view their cart, and complete payments.


## Related Ticket(s)


## Steps to Test
-Add and commit on your branch
-Pull down sb-product-details
-Run your server
-Click on the "View Cart" link in the nav bar. If you don't have an open order, you should be redirected to a page saying the cart is empty.
-Navigate to the details page of a product and click "add to cart" - you should be redirected back to the index view of products. - if this has opened a new order, it should have posted to your database, and a post should be made to OrderProduct in the database regardless of whether this created a new order
-If you click View Cart now, your items should be in the cart (with the title displayed)
-If you click the "Complete Payment" link at the bottom, you should be taken to a page where you can select a payment type from the drop down menu
-Upon selecting a payment type and clicking the "Complete Payment" button, you will be redirected to a page with a confirmation message - The order should not be updated in the database with the completion date and PaymentTypeId filled out. The products purchased should have their quantity decremented in the database by the number purchased.



# Checklist:
